src/layer/S3OLCILayer.ts

```diff
@@ -1,7 +1,9 @@
 import moment from 'moment';
 
 import { BBox } from 'src/bbox';
-import { PaginatedTiles, Link, LinkType } from 'src/layer/const';
+
+import { PaginatedTiles, Link, LinkType, RequestConfiguration } from 'src/layer/const';
+
```

Comment:
Very very very very minor nitpick: newlines are tipically used to distinguish between system imports (from external packages) and local imports, so it makes sense to remove them here. It's more a styling rule though, fftr either way.

EDIT: same below in S3SLSTR and S5P.